### PR TITLE
feat: filter partner

### PR DESCRIPTION
**Pull request by ArinBisa123**

### CODE REVIEW

File: src/components/Logbook/SearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const SearchBar = ({ onSearch, isOpenSide }) => {
+  const handleSearchChange = (e) => {
+    onSearch(e.target.value);
+  };
+  return (
+    <div className="flex w-full">
+      <div
+        className={`${
+          isOpenSide ? "w-[300px]" : "w-[400px]"
+        } h-[32px] border flex px-[12px] rounded-l-[6px]`}
```
Comment (iqbalalayubbi) [inline src/components/Logbook/SearchBar.jsx:11]: `h-8 px-3 rounded-l-md`

File: src/components/Logbook/SearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      >
+        <input
+          type="text"
+          placeholder="Search"
+          aria-label="Search"
+          className="w-full focus:border-none border-none focus:outline-none"
+          onChange={handleSearchChange}
+        />
+      </div>
+      <div className="w-[32px] h-[32px] rounded-r-[6px] bg-[#1677FF] flex justify-center items-center">
```
Comment (iqbalalayubbi) [inline src/components/Logbook/SearchBar.jsx:21]: use `<Flex>` component

File: src/components/Logbook/SearchBar.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <SearchOutlined className="text-[16px] text-white" />
```
Comment (iqbalalayubbi) [inline src/components/Logbook/SearchBar.jsx:22]: `text-base`

**Review by iqbalalayubbi — CHANGES_REQUESTED** [review]

File: src/pages/LogbookMentor/LogbookMentorDashboard.jsx [hunk 4]
```diff
@@ -6,12 +6,12 @@ import {
   LogbookMentorHeader,
   LogbookMentorPagination,
 } from "./libs/components/component";
-import { Flex } from "antd";
+import { Breadcrumb } from "antd";
```
Comment (iqbalalayubbi) [inline src/pages/LogbookMentor/LogbookMentorDashboard.jsx:9]: import from `@/components/components`

File: src/pages/LogbookMentor/LogbookMentorDashboard.jsx [hunk 5]
```diff
@@ -36,67 +36,51 @@ const LogbookMentorDashboard = () => {
...
-        <span className="text-3xl font-medium">Logbook Mentor</span>
-        <Flex justify="between" align="center" gap={5}>
-          <span className="text-colorTextDescription">Home</span>
-          <span className="text-colorTextDescription">/</span>
-          <span className="text-colorPrimaryTextActive">Logbook Mentor</span>
-        </Flex>
-      </Flex>
+    <section>
+      {/* HEADER */}
+      <div className="bg-white flex items-center justify-between py-4.5 px-10.5 rounded-lg shadow-[0px_4px_4px_0px_rgba(0,0,0,0.10)]">
```
Comment (iqbalalayubbi) [inline src/pages/LogbookMentor/LogbookMentorDashboard.jsx:45]: prefer use `<Flex>` component

**Review by iqbalalayubbi — CHANGES_REQUESTED** [review]

**Review by iqbalalayubbi — COMMENTED** [review]
I changed some import styles on logbook and logbook mentor. You can check it again and make sure it work fine

**Comment by ArinBisa123** [comment]
@iqbalalayubbi i already check it, everything works fine

**Review by iqbalalayubbi — APPROVED** [review]

**Pull request merged**